core/arch/arm/include/arm64.h

```diff
@@ -355,6 +355,14 @@ static inline __noprof void tlbi_vale1is(uint64_t va)
 	asm volatile ("tlbi	vale1is, %0" : : "r" (va));
 }
 
+static inline void write_64bit_pair(uintptr_t dst, uint64_t hi,
```

Comment:
Please replace `uintptr_t` with `uint64_t`. Functions in this file are supposed to be as primitive as possible and the assembly instruction expects a 64-bit register.

Reply:
Done